api/lib/mno_enterprise/concerns/controllers/jpi/v1/impac/kpis_controller.rb

```diff
@@ -56,14 +56,14 @@ def create
       authorize! :manage_dashboard, dashboard
     end
     # TODO: nest alert in as a param, with the current user as a recipient.
-    @kpi = kpi_parent.kpis.create(kpi_create_params)
-    unless kpi.errors?
+    @kpi = MnoEnterprise::Kpi.create(kpi_create_params)
+    if @kpi.errors.empty?
       # Creates a default alert for kpis created with targets defined.
       if kpi.targets.present?
-        current_user.alerts.create({service: 'inapp', impac_kpi_id: kpi.id})
+        MnoEnterprise::Alert.create({service: 'inapp', kpi_id: kpi.id, recipient_ids: [current_user.id]})
```

Comment:
It is in the recipient ids indeed, it was done this way on mnohub v1. I've simplified.